src/main/java/lama/Lama.java

```diff
@@ -0,0 +1,63 @@
+package lama;
+
+import lama.command.Command;
+
+/**
+ * Represent the main entry point of the Lama application.
+ * Handles initialisation of components, the main program
+ * loop and process user command.
+ */
+public class Lama {
+
+    private Storage storage;
+    private Ui ui;
+    private TaskList taskList;
+
+    /**
+     * Construct a Lama object with specified file path for storing data.
+     * Initialise the UI, storage and task list.
```

Comment:
Should this be "constructs" and "initialises" instead to comply with the coding standard?